fence/blueprints/data/blueprint.py

```diff
@@ -89,19 +89,18 @@ def upload_data_file():
                 authz_err_msg.format("create' and 'write-storage", authz)
                 + " Falling back to 'file_upload' on '/data_file'."
```

Comment:
```suggestion
                authz_err_msg.format("create' and 'write-storage", authz)
```
not falling back on that anymore